src/main/java/com/artipie/adapters/docker/DockerPermissions.java

```diff
@@ -32,7 +32,7 @@ public DockerPermissions(final Permissions origin) {
     }
 
     @Override
-    public boolean allowed(final Authentication.User user, final String action) {
+    public boolean allowed(final AuthUser user, final String action) {
```

Comment:
I don't like this `DockerPermissions` class.
It looks like bottleneck, and we should get rid of this class later.

Reply:
We will get rid of it as soon as we implement docker permissions in docker adapter.